chain/chain.go

```diff
@@ -18,6 +18,9 @@ var (
 
 	// GenesisDifficulty is the default difficulty of the Genesis block.
 	GenesisDifficulty = big.NewInt(131072)
+
+	// DefaultBlockGasTarget is the default value for the block gas target for new blocks
+	DefaultBlockGasTarget uint64 = 8000000
```

Comment:
Remove the DefaultBlockGasTarget.

If the server doesn't have the block gas target set, then just copy over the limit from the parent block when it is proposing a block.

Reply:
I think that having default block gas target would cause 2 issues:

1. It would confuse users because they didn't know about it and suddenly they see the limit increasing to the default value from the initial gas limit in the genesis.
2. If I want to have a permissioned network with an unchanging block limit, then I need to set the initial block gas limit in the genesis, and set the additional flag in each server command that I run.

Reply:
This is completely correct, and both are valid points 👍 

I overlooked the usability aspect of this, as it would be frustrating to have the chain move in a random direction even though the genesis value has been set.

Fixed 👍 

a8b0343